web/next.config.js

```diff
@@ -136,195 +136,1944 @@ const nextConfig = {
 		];
 	},
 	async redirects() {
+		// TODO the following redirects are currently commented out in the array at correct specificity level:
+		// ------------------------------------------------------------
+		//   1. SOURCE: /get-involved/our-committees/join-a-committee/nice-committee-vacancies-for-patients--service-users--carers-and-lay-people
+		//      DESTINATION: n/a stay in Orchard until we figure out the TRAC embed
+		// ------------------------------------------------------------
+		//   2. SOURCE: /get-involved/our-committees/join-a-committee/nice-committee-vacancies-for-health--social-care-and-other-professionals-and-practitioners
+		//      DESTINATION: n/a stay in Orchard until we figure out the TRAC embed
+		// ------------------------------------------------------------
+		//   3. SOURCE: /get-involved/our-committees/join-a-committee/apply-to-be-a-chair-on-a-nice-committee
+		//      DESTINATION: n/a stay in Orchard until we figure out the TRAC embed
+
 		return [
+			// === BLOCK GENERATED FROM EXCEL MIGRATION PHASE 2 ===
 			{
-				source: "/news/nice-newsletters-and-alerts",
-				destination: "/nice-newsletters-and-alerts",
+				source:
+					"/about/what-we-do/our-programmes/nice-guidance/about-technology-appraisal-guidance/participation-in-clinical-trials-and-subsequent-access-to-drugs-appraised-by-nice-a-statement-on-the-applicability-of-about-technology-appraisal-guidance",
+				destination:
+					"/position-statements/participation-in-clinical-trials-and-subsequent-access-to-drugs-appraised-by-nice-a-statement-on-the-applicability-of-technology-appraisal-guidance",
 				permanent: true,
-			},
+			}, // Generated from Excel Migration Phase 2 URLs
```

Comment:
Could we remove this repeated 'generated from ...' comment (and the others like it) as its making readability difficult 😖

Reply:
The order of the list also feels quite/very hard to follow, can we rationalise this?

Reply:
That was so I could keep track of which redirects came from where in the config, but sure.